Gemfile

```diff
@@ -68,7 +68,6 @@ gem 'mimemagic'
 gem 'acts-as-taggable-on', '~> 7.0'
 # this doesn't install cleanly on a Mac
 # We aren't currently using this anyway
-gem 'seven_zip_ruby'
```

Comment:
@ZachCMP didn't we solve this for you?  (we should add it back either way)

Reply:
@eanders Yup, solved